src/pages/UsersTable/UsersTable.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react'
```

Comment:
Pelo jeito esse componente não é utilizado em lugar nenhum da aplicação. Se não usa, remove. E se for usar, move para a pasta de componentes.